_data/current.yml

```diff
@@ -363,6 +363,12 @@
   twitter: GermanTesting
   status: Early Bird Pricing open until March 13, 2020
 
+- name: TestingUY 2020
+  location: Montevideo, Uruguay
+  dates: "May 11-12, 2020"
+  url: http://testinguy.org
```

Comment:
Generally I add the `utm_source` tracking code to each url for better tracking for the conference organizer. I can add this later though